collective/js/galleria/resources/galleria.min.js

```diff
@@ -0,0 +1,282 @@
+(function(window,factory){window.Galleria=factory(window,window.jQuery);}(window,function factory(window,$,Galleria,undef){var doc=window.document,$doc=$(doc),$win=$(window),jQuery=$,protoArray=Array.prototype,VERSION=1.61,DEBUG=true,TIMEOUT=30000,DUMMY=false,NAV=navigator.userAgent.toLowerCase(),HASH=window.location.hash.replace(/#\//,''),PROT=window.location.protocol=="file:"?"http:":window.location.protocol,M=Math,F=function(){},FALSE=function(){return false;},MOBILE=!((window.screen.width>1279&&window.devicePixelRatio==1)||(window.screen.width>1000&&window.innerWidth<(window.screen.width*.9))),IE=(function(){var v=3,div=doc.createElement('div'),all=div.getElementsByTagName('i');do{div.innerHTML='<!--[if gt IE '+(++v)+']><i></i><![endif]-->';}while(all[0]);return v>4?v:doc.documentMode||undef;}()),DOM=function(){return{html:doc.documentElement,body:doc.body,head:doc.getElementsByTagName('head')[0],title:doc.title};},IFRAME=window.parent!==window.self,_eventlist='data ready thumbnail loadstart loadfinish image play pause progress '+
```

Comment:
@cleberjsantos  your `galleria.min.js` is bigger than GalleriaJS 1.6.1:

https://github.com/GalleriaJS/galleria/blob/1.6.1/dist/galleria.min.js

It was you who minified?

Reply:
@wesleybl If you compare the original js with previous versions, you will also see the same result. For the Gallery to work in Plone we need a patch apply in JS.

Reply:
I understand that patches are needed. What caught my attention is that, despite the patches being few lines, the minified file here was 92 KB and the original file is 71 KB. The minifier there seems to be more efficient than the one used here. 

Maybe it's better to change the file in a galleria clone and use process of their to minify.

I am going to try.

Reply:
+1

Reply:
@cleberjsantos I used the GalleriaJS repository to generate the file galleria.min.js, and the resulting file was much smaller.

To see:

https://github.com/collective/collective.js.galleria/pull/8/commits/82e818c17ff9d854407e60dacbaba4fcb9100170